devdoc/DevTestWithLDAP.md

```diff
@@ -0,0 +1,146 @@
+- `systemBasedn` is the root of the DS tree used by the platform. It must match the backend suffix used in step #3 when configuring the 389ds server.
+- `systemPassword` must match the 389ds `DS_DM_PASSWORD` configured in the `dockerfiles/examples/ldap/docker-compose.yaml` file.
+
+## Add Secret for HPCC Admin User
+
+Next you must add the `myhpccadminsecretkey` to your HPCC platform deployment. If there are concerns about storing these credentials on disk you should remove this secret directory when not in use, or manage it with an encryption/decryption utility such as `gocryptfs`
+
+First note the root of your platform install location, which we'll refer to as `<HPCC_ROOT>`. Package installs will be rooted in `/` but dev installs are typically located at `$HOME/runtime`.
+
+1. Create all directories in this path if they don't already exist:
+
+    ```
+    <HPCC_ROOT>/opt/HPCCSystems/secrets/authn/myhpccadminsecretkey
+    ```
+2. Inside `myhpccadminsecretkey` create two text files with no extra whitespace and no terminal newline:
+    - A file named `username` containing `hpcc_admin`
+    - A file named `password` containing the password you want the hpcc admin user to have. We'll refer to it as `<hpcc_admin_pw>`
+
+## Run and Use Platform
+
+1. Start up the platform:
+
+    ```bash
+    hpcc-init start
+    ```
+2. Login to ECL Watch at [http://127.0.0.1/8010](http://127.0.0.1/8010) using the credentials:
+    - user: `hpcc_admin`
+    - password: `<hpcc_admin_pw>` (that you added to the secret above)
+
+Initially your admin will only have SmcAccess, so you may see some access failure warnings, but you can navigate to the Topology | Security tab to customize the HPCCAdministrators permissions and add any other users, groups or permissions needed for testing.  These warnings could also be due to permissions caching, and may be resolved after a platform restart.
+
+# Containerized Platform + Docker LDAP
+
+This containerized deployment enables ldap auth for ECL Watch only, but it can be extended easily to other ESP services.
+
+## Create k8s Secrets for Platform
+
+You'll need two k8s secrets, one for the HPCC administrator and another for the LDAP server administrator. Once created they'll persist in the Kubernetes backing store.
+
+1. Create secret for HPCC administrator:
+    ```bash
+    kubectl create secret generic myhpccadminsecretkey --from-literal=username=hpcc_admin --from-literal=password=<hpcc_admin_pw>
+    ```
+1. Create secret for the LDAP server administrator:
+    ```bash
+    kubectl create secret generic admincredssecretname --from-literal="username=Directory Manager" --from-literal=password=<directory_manager_pw>
+    ```
+
+## Customize Platform Helm Values
+
+We'll be using the customized helm values file at `helm/examples/ldap/hpcc-values.yaml`. It will run all standard ESP services, but it uses ldap authentication only for the eclwatch service. Customize this file further as needed.
```

Comment:
Helm should be proper case in this instance: 
We'll be using the customized helm values file...
s/b
We'll be using the customized Helm values file...